### PR TITLE
enhance build-jobs

### PR DESCRIPTION
**Pull request by chalstrick**

### CODE REVIEW

File: jenkins-job-generator/definitions/linux-package-jobs.yml [hunk 1]
```diff
@@ -167,38 +166,44 @@
...
-    description: 'Recreate debian repository.'
+- project:
+    name: resign-debian-repository
+    platform: *platforms
+    jobs:
+        - resign-debian-repository-{platform}
+
+- job-template:
+    name: resign-debian-repository-{platform}
+    description: 'Resign debian repository on {platform}.'
```
Comment (RealCLanger) [inline jenkins-job-generator/definitions/linux-package-jobs.yml:177]: Here should be an entry: `defaults: 'linux_package_global'`, like in the other template above.

File: jenkins-job-generator/definitions/linux-package-jobs.yml [hunk 2]
```diff
@@ -167,38 +166,44 @@
...
     project-type: pipeline
     concurrent: false
```
Comment (RealCLanger) [inline jenkins-job-generator/definitions/linux-package-jobs.yml:179]: Please also add `sandbox: true`, as above in the file.

**Review by RealCLanger — CHANGES_REQUESTED** [review]
Looks good overall. I made a few comments. Please also adapt views.yml to reflect the new resign jobs.

File: jenkins-job-generator/definitions/linux-package-jobs.yml (hunk 1 above)
Comment (chalstrick) [reply jenkins-job-generator/definitions/linux-package-jobs.yml:177]: done

**Review by RealCLanger — APPROVED** [review]
Looks good now. Go for it. Please squash & merge.

**Pull request merged**